src/webgpu/shader/execution/builtin/builtin.ts

```diff
@@ -18,13 +19,50 @@ export interface Comparison {
   expected: string; // The string representation of the 'expected' value (possibly with markup)
 }
 
+/** FloatTest is a function that compares whether the two floating point numbers match. */
+export interface FloatTest {
```

Comment:
nit: Maybe this should be called FloatMatch ?

Reply:
Done